examples/cifar_cnntransformer/README.md

```diff
@@ -3,11 +3,14 @@
 ### 1. Description
 In this example we train a standard 8 layer CNN on CIFAR10 as a baseline. We then take the same CNN architecture and stack a Transformer-Encoder ontop and train this new CNNTransformer model from scratch. We present several different variants of this model where we only alter the Transformer size. Below, the validation accuracy of each model is compared.
 
-![Model Comparisons](CIFAR10-ModelComparison-ValAcc.png)
+[//]: # (![Model Comparisons]&#40;CIFAR10-ModelComparison-ValAcc-Lightning.png&#41;)
+<div align="center">
+	<img src="CIFAR10-ModelComparison-ValAcc-Lightning.png" alt="Editor" width="80%">
+</div>
 (The code example on this page does not aim to show any meaningful results, but only demonstrate the use of the CNNTransformer)
```

Comment:
Suggest change: show any meaningful results --> achieve state-of-the-art results

Should we have a full stop in the end?

Reply:
Change made. I have also changed the wording from "code example on this page" to "code example in this folder" for better clarification.

Reply:
Changes made unsatisfactory. Please see new comments.